nexus/src/external_api/http_entrypoints.rs

```diff
@@ -7386,13 +7390,49 @@ async fn user_list(
     apictx.external_latencies.instrument_dropshot_handler(&rqctx, handler).await
 }
 
+/// List users
+#[endpoint {
+    method = GET,
+    path = "/v1/users",
+    tags = ["silos"],
+}]
+async fn user_list_v1(
+    rqctx: RequestContext<Arc<ServerContext>>,
+    query_params: Query<PaginatedById<params::OptionalGroupSelector>>,
+) -> Result<HttpResponseOk<ResultsPage<User>>, HttpError> {
+    let apictx = rqctx.context();
+    let nexus = &apictx.nexus;
+    let query = query_params.into_inner();
+
+    let pag_params = data_page_params_for(&rqctx, &query)?;
+    // let scan_params = ScanById::from_query(&query)?;
```

Comment:
I minor thing, but move all this into the handler definition. We're doing that because timing data is only recorded for things inside the handler so we want as much as possible to be grouped there.